Scripts/NuGet/EdFi.Installer.SandboxAdmin/EdFi.Installer.SandboxAdmin.nuspec

```diff
@@ -17,6 +17,6 @@
     <file src="Ed-Fi-Common/**" />
     <file src="Ed-Fi-ODS/**" />
     <file src="Ed-Fi-ODS-Implementation/**" />
-    <file src="../../../LICENSE.txt" />
+    <file src="../shared/LICENSE.txt" />
```

Comment:
why are we importing from a shared folder, when the file is in the root?

Reply:
Because our directory structure is not well-suited to having appropriate checkout rules that would allows us to trigger builds appropriately when relying on the file in the root directory. See [Tech Docs](https://techdocs.ed-fi.org/display/EDFIODS/ODS+Build+Configurations) for lengthier explanation.